### ISSUE TITLE
Recruit state participants for the pilot

### ISSUE THREAD

**Issue by lauraponce** [18F/cms-hitech-apd#1311]
MeT will recruit state participants by sending an email out to the all states listserv. Once the email is sent, a meeting should be scheduled with the team (MeT, Nick, Jerome, Laura + rest of the 18F team as optional) to evaluate the states who volunteered against the selection criteria to make a decision.

### This task is done when...
- [x] Recruiting email has been sent to the all states listserv (#1312)
- [x] 4 state participants have been selected based on criteria in #1576 
- [x] Selected states have been notified that they have been selected and they have confirmed (#1313)
- [x] Non-selected states have been notified (#1313)

**Comment by lauraponce** [comment]
A few questions/updates to help move this forward:

1. The [recruiting email](https://docs.google.com/document/d/1ktgEMakVf73pMNp3bSHjBlP-lwhFNe3ruAcz4BRGkyo/edit?usp=sharing) is drafted, the only thing that needs updating in it is all the dates to match what we're aiming for currently.

2. To prepare for sending it and to set expectations in the recruiting message, the dates that need to be nailed down are: 
- When to start recruiting and send the email
- When do we want states to respond by (the cut-off for recruiting)
- When do states need to submit their APDs by
- Rough date for when they will be able to log in and start using the eAPD
- Rough date for the interest meeting (nice to have)

3. It would be helpful to know the status of #1481 so that the date for the interest meeting can be included in the recruiting email but it's not essential to have it in there so it's okay if it's still uncertain. @MCorradoHMA @CSwartzHMA do you have any updates on that one?

4. Does it make sense to reassign this issue to @NAretakis and/or @jeromeleecms since one of you will be hitting the send button from your shared CMS email account?

5. @nkkl do you have the list of states on the distro list? It would be helpful to know whose emails we have so we know who is missing.

6. I'm assuming we'll want to send the recruiting email to all states and not just the ones who signed up to be added to the email list — @NAretakis @jeromeleecms what are your thoughts on that?

**Comment by jeromeleecms** [comment]
Pulling this back after our tabletop exercise discussion for next sprint. Will assign this back to MeT at our next sprint planning on 06/06 to start the recruitment process.

**Comment by lauraponce** [comment]
Recruiting email from the [pilot communications doc](https://docs.google.com/document/d/1ktgEMakVf73pMNp3bSHjBlP-lwhFNe3ruAcz4BRGkyo/edit#):

> Hello!
> 
> As you may know, CMS, MeT, and 18F (GSA) have been working together to modernize the HITECH IAPD creation and submission process. The new eAPD will be an online application (app) that helps states submit IAPDs and associated contracts easily and will help CMS analysts review them more efficiently. 
> 
> Building on the great work and feedback received to date, we are ready to take the next step. We are seeking four states to volunteer to pilot the eAPD by using it to prepare your upcoming IAPD submission to CMS. We are particularly seeking states whose IAPDs are due for submission on or before August 1.
> 
> If you are interested, please email CMS-EAPD@cms.hhs.gov by 5/28.
> 
> As part of the development of the eAPD, we have conducted numerous usability sessions to get input from states on the design and features of the app. Thanks to those states, the eAPD has been designed to address many of the common issues with the current IAPD submission process.
> 
> As with states that have previously volunteered to provide feedback, pilot states in this round will have the opportunity to have a direct impact on the changes made to the app before nationwide release.
> 
> We will also hold an interest meeting for states to learn more about eligibility, commitment, and timeframes on 5/24. Thank you for your time and support as always.

**Comment by lauraponce** [comment]
Confirmation email from the pilot communications doc: 

> Hi NAME,
> 
> We’re excited to have you join us in the eAPD pilot as one of the four state participants!
> 
> **What’s next**
> 
> We will create an account for you and will send over those details along with the URL where you can log into the eAPD application. The estimated timeline for when you’ll be able to log in and start filling in your APD is a few weeks, so we expect that you’ll be able to start using the app around the beginning of June. We’ll be sure to keep you posted with any updates during this timeframe.
> 
> **How the eAPD process will work** 
> 
> You’ll use the app to fill in your APD just like you normally would. At any point in time, you can export a PDF copy of the information you’ve entered, but any changes that you need to make should be done through the app. Once you’re ready to submit to CMS for approval, you’ll export the final version of your APD and will email it to your CMS reviewer (you’ll find more detailed instructions about how to do this within the app).
> 
> Please submit your APD by AugustJuly 1 to ensure there is enough time for CMS to respond to you with any questions.
> 
> **Sharing feedback**
> 
> We’ll be scheduling three sessions with you to see how your eAPD experience is going and to collect your feedback. 
> 
> The first session will be for shadowing you while you fill in your APD — once you’re able to log in and get started, we’ll find the time that works best for you to do this with us. The second session will be an interview format where we’ll ask some questions about your experience after you submit your APD to CMS. The third session will be a lightweight follow-up interview after your APD has been approved.
> 
> If you have feedback that you’d like to share asynchronously while it’s still fresh in your mind, please add your thoughts here: https://forms.gle/1QYMduhWNWyUem5F6 — use this form at any time to provide feedback as often as you’d like, but please don’t input sensitive data.
> 
> If you run into any issues, you can reach out to us at CMS-EAPD@cms.hhs.gov. 
> 
> We’ll reach out to you with any updates to the pilot and will be in touch. Thanks again for your participation!

**Comment by lauraponce** [comment]
Follow-up email to states who were interested but not selected to participate (from pilot communications doc):

> Hello!
> 
> Thank you for your interest in the pilot. The participant slots have all been filled and the four states that will be participating have been notified. We selected the states based on a few factors including the complexity of their APD and their timeline.
> 
> After the pilot is over in the next few months, we’ll be making updates to the application based on the feedback. If you would like to share your thoughts or have any questions, reach out to us at CMS-EAPD@cms.hhs.gov. 
> 
> We will share updates and timelines for the production-ready eAPD application in the near future. Thanks again!

**Comment by lauraponce** [comment]
I posted the content here since MeT doesn't have access to the Google Doc. @jeromeleecms can you give these a once over too? I _think_ I edited all the dates correctly and there might be some additional changes to the framing that we want to make so having a second set of eyes on this would be good!

**Comment by jeromeleecms** [comment]
I updated the Google doc with some minor updates + one suggestion. MeT is planning on review these as well before our Monday distribution. Do you want to do a quick 15 minute session with them to do a final read through?

**Comment by lauraponce** [comment]
Sounds good! Maybe we could use the first 15 of our coworking time today to review?

**Comment by MCorradoHMA** [comment]
Updating the email to be sent today...I asked Benita to set up the webinar so she can update the placeholder with the link. @jeromeleecms @NAretakis please take a final review and once you give the OK I'll send to Benita to email out. (cc @CSwartzHMA @lauraponce  for your review as well).

~~~~~~~~~~~~~
Hello!

As you may know, CMS, MeT, and 18F (GSA) have been working together to modernize the HITECH IAPD creation and submission process. The new eAPD will be an online application (app) that helps states submit IAPDs and associated contracts easily and will help CMS analysts review them more efficiently.

Building on the great work and feedback received to date, we are ready to take the next step. We are seeking four states to volunteer to pilot a component of the eAPD app - the eAPD Builder - by using it to prepare your upcoming IAPD submission to CMS. 

If you are interested, please email CMS-EAPD@cms.hhs.gov by 6/29.

As part of the development of the eAPD, we have conducted numerous usability sessions to get input from states on the design and features of the app. Thanks to those states, the eAPD has been designed to address many of the common issues with the current IAPD submission process.

As with states that have previously volunteered to provide feedback, pilot states in this round will have the opportunity to have a direct impact on the changes made to the app before nationwide release.

We will also hold an interest meeting for states to learn more about eligibility, commitment, and timeframes on 6/24 from 4-5PM ET [INSERT LINK]. Please join us if you'd like to learn more.

Thank you for your time and support as always.

**Comment by MCorradoHMA** [comment]
One last question discussed with @jeromeleecms on friday...do you want people to sign up through the cms-eapd email or would you like them to email the MeT to sign up? I'm happy to track interest but I don't get the CMS-EAPD email. You can forward any responses to me or we can use a different email. @NAretakis and @jeromeleecms let me know what you think works best for you.

**Comment by jeromeleecms** [comment]
@MCorradoHMA Let's have the state's respond to MeT and have you all track the people we need to invite to the interest meeting. If you want to send @NAretakis  and I the list after the first week and then after the reminder email (right before the interest meeting), that would be great!

**Comment by lauraponce** [comment]
[Pilot communications.docx](https://github.com/18F/cms-hitech-apd/files/3273432/Pilot.communications.docx)

**Comment by MCorradoHMA** [comment]
recruiting email sent 6/10!

**Comment by MCorradoHMA** [comment]
Sign up update as of EOD 6/10. The following states emailed and said they were interested:

-	New Mexico (but they want to know if this can be up and running for a submission and receive approval for funding to start Oct. 1)
-	Alaska (they have a contractor that develops their IAPDs)
-	Michigan
-	Wyoming
-	Oregon (possibly – they want to know if they can use it for this year’s submission or just a test; also they wanted to know about the time commitment – I can make sure they attend the webinar on 6/24)
-	Alabama

**Comment by MCorradoHMA** [comment]
Sign-up update as of 6/24 4pmET (prior to the informational webinar)

-	West Virginia
-	Indiana
-	Delaware 
-	Florida
-	California
-	Louisiana
-	Oklahoma (but I think they are submitting an APD by July 3)
-	Colorado
-	Virginia
-	Iowa
-	Idaho
-	New Jersey
-	Puerto Rico
-	Kentucky
-	Michigan
-	Wyoming
-	New Mexico
-	Alaska
-	Alabama
-	Oregon

**Comment by NAretakis** [comment]
@MCorradoHMA is there a final list of states, post-webinar?

**Comment by MCorradoHMA** [comment]
Here it is!

States that signed up and confirmed they wanted to participate after the webinar was held:
Nevada
Alaska
Missouri
Iowa
Wyoming
New Jersey
Oregon
New Mexico
Idaho
Hawaii
Delaware
Tennessee
Maine
Florida
Louisiana
Georgia
West Virginia

Other interested states that didn’t confirm again:
Indiana
California
Colorado
Virginia
Puerto Rico

**Comment by MCorradoHMA** [comment]
Email was sent to FL, ID, AK, and DE (alpha selection) to confirm they have Chrome. As of now, FL and ID confirmed yes. Waiting on AK and DE.

**Comment by lauraponce** [comment]
That's great news! Let me know if you need help tracking down any of the drafts for email communications or anything since they are in Google Drive, they should also be in GitHub as well.

**Comment by NAretakis** [comment]
@MCorradoHMA  any news from AK and DE?

**Comment by CSwartzHMA** [comment]
AK and DE confirmed yes, as well.

**Comment by MCorradoHMA** [comment]
All confirmed they have access to Chrome. 
Email to be sent out 7/9 to selected and non-selected states.

Issue closed!

**Comment by jeromeleecms** [comment]
@MCorradoHMA Are we ready to close this yet?

**Comment by MCorradoHMA** [comment]
@jeromeleecms I think it's fine to close...technically the 4 states were supposed to confirm before we could close it out but we actually did not specifically ask them to confirm. So let's close it and if they do not log in with their credentials we can follow up!
